backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -422,96 +418,94 @@ export class AnalyticsRepository {
     skip: number,
     take: number
   ): Promise<CSVExportDataRow[]> {
+    const individualEnrollmentRepository = getCustomRepository(IndividualEnrollmentRepository, 'export');
```

Comment:
@ppratikcr7 This query can further be simplified if we don't get the experiment detail in the query. You can query the experiment detail separately and use the paginated query to do the groupBy part and populate the details on the server side.